img2mol/inference.py

```diff
@@ -39,14 +46,36 @@ class Img2MolInference(object):
     """
     Inference Class
     """
-    def __init__(self, model_ckpt: Optional[str] = None, device: str = "cpu"):
+    def __init__(
+        self,
+        model_ckpt: Optional[str] = None,
+        device: str = "cuda:0" if torch.cuda.is_available() else "cpu",
+        local_cddd: bool = False
+    ):
         super(Img2MolInference, self).__init__()
+        if local_cddd:
+            self.cddd_inference_model = CDDDInferenceModel()
+        else:
+            self.cddd_inference_model = False
+            self.cddd_server = CDDDRequest()
```

Comment:
```python
        if local_cddd:
            self.cddd_inference_model = CDDDInferenceModel()
            self.cddd_server = None
        else:
            self.cddd_inference_model = None
            self.cddd_server = CDDDRequest()
```
It's good practice to not have conditionally created attributes in instances and `None` is a better default for not set than `False` here.

Reply:
I have changed it back to the old behaviour where the CDDDRequest object is fed to __call__ instead of instantiating it manually. This way, we don't break anyone's code, and it works exactly the way it worked before.
Instead, only the CDDD model is loaded if the local_cddd parameter is set to `True`:
```
if local_cddd:
            self.cddd_inference_model = CDDDInferenceModel()
        else:
            self.cddd_inference_model = None
```